src/theme.tsx

```diff
@@ -15,6 +17,30 @@ export const theme = createTheme({
   }
 });
 
+export const greenTheme = createTheme({
+  typography: {
+    fontFamily: '"Inter", sans-serif'
+  },
+  palette: {
+    primary: {
+      light: green[100],
+      main: green[400],
+      dark: green[600]
+    },
+    secondary: {
+      light: gray[100],
+      main: gray[400],
+      dark: gray[600]
+    },
+    warning: {
```

Comment:
Purple is our accent colour, so I don't think it should be used for warning. I have defined the semantics now - error, warning, success and info